oci/pull.go

```diff
@@ -102,7 +104,17 @@ func (o *OrasRemote) PullPath(ctx context.Context, destinationDir string, desc o
 		return err
 	}
 
-	return os.WriteFile(fullPath, b, helpers.ReadWriteUser)
```

Comment:
to ensure 1:1, we should set these same file permissions on the file

Reply:
Yeah, nice catch